salsah/src/public/js/imagebase.js

```diff
@@ -90,6 +90,10 @@ $(function() {
 			//***************
 
 
+			var sortedprops = Object.keys(resource.props).map(function(propname) {return [propname, resource.props[propname]];});
```

Comment:
Could you add some documentation describing the data structure you are creating here?

Probably you could also create an Array of objects instead of a 2-dimensional Array. You could use keys `propname` and `value`.